shared/config/config.go

```diff
@@ -65,6 +65,9 @@ type FundRebalancerConfig struct {
 	// before a rebalance is triggered to move uusdc from other chains to this
 	// chain.
 	MinAllowedAmount string `yaml:"min_allowed_amount"`
+	// RebalanceTransferTimeout is the maximum amount of time to wait before marking
+	// a pending rebalance transfer status as abandoned
+	RebalanceTransferTimeout *time.Duration `yaml:"rebalance_transfer_timeout"`
```

Comment:
IMO we do not need to surface this configuration to end users. We can just set in opinionated constant in the code of something like 10 minutes.